database/changesets/20210423_AddSinkSourceExecutorColumnsToWorkload.xml

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.1" encoding="UTF-8" standalone="no"?>
```

Comment:
Just sanity checking, since you got to remove a number of changesets already… is this one still needed?

Reply:
I removed the changeset and reran the integration tests without a problem, so I think we can remove it. I've done so. Great catch!